recipes/gensim/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set name = "gensim" %}
+{% set version = "2.3.0" %}
+{% set sha256 = "7d0dccc7d2c576e270037949874800b7cfbc86ef081ff981483f612cd18e223f" %}
```

Comment:
Yes, We only need to update the ``version`` and the ``sha256`` for every release. This is done by sending a PR to the complete gensim feedstock repository.